src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -177,15 +177,38 @@ export async function POST(request: NextRequest) {
   // reference example events: https://github.com/mozilla/fxa/blob/main/packages/fxa-event-broker/README.md
   for (const event in decodedJWT?.events) {
     switch (event) {
-      case FXA_DELETE_USER_EVENT:
+      case FXA_DELETE_USER_EVENT: {
         logger.info("fxa_delete_user", {
           subscriber: subscriber.id,
           event,
         });
 
+        // get profile id
+        const result = await getOnerepProfileId(subscriber.id);
+        const oneRepProfileId = result?.[0]?.["onerep_profile_id"] as number;
```

Comment:
Would you mind filing a bug to refactor this? It'd be nice if the caller just got a `number` back.

Reply:
I think I can just refactor that in this PR

Reply:
actually let me open another PR for it..

Reply:
https://mozilla-hub.atlassian.net/browse/MNTOR-2874